test/cli.test.js

```diff
@@ -0,0 +1,42 @@
+var should = require('should'),
+  cli = require('../bin/cli'),
+  theme = {
+    results: {
+      error: [],
+      warning: [],
+      recommendation: []
+    }
+  };
+
+process.env.NODE_ENV = 'testing';
```

Comment:
Redundant, because of https://github.com/TryGhost/gscan/blob/master/package.json#L32.
The confusion probably came from [this file](https://github.com/TryGhost/gscan/blob/master/test/general.test.js#L16) - this can be removed as well.